File: src/test/java/org/raku/comma/parsing/LoopLexerBugTest.java
package org.raku.comma.parsing;

import com.intellij.testFramework.fixtures.BasePlatformTestCase;
import org.raku.comma.filetypes.RakuScriptFileType;

public class LoopLexerBugTest extends BasePlatformTestCase {
    public void testLexerBug1() {
        myFixture.configureByText(RakuScriptFileType.INSTANCE, "loop (my $a = 5; $a < <caret>)");
        myFixture.type("5Foo");
        myFixture.checkResult("loop (my $a = 5; $a < 5Foo)");
    }
}

File: src/test/kotlin/org/raku/comma/parsing/LoopLexerBugTest.kt
package org.raku.comma.parsing

import com.intellij.testFramework.fixtures.BasePlatformTestCase
import org.raku.comma.filetypes.RakuScriptFileType

class LoopLexerBugTest : BasePlatformTestCase() {
    fun testLexerBug1() {
        myFixture.configureByText(RakuScriptFileType.INSTANCE, "loop (my \$a = 5; \$a < <caret>)")
        myFixture.type("5Foo")
        myFixture.checkResult("loop (my \$a = 5; \$a < 5Foo)")
    }
}
